src/io/flutter/view/InspectorPanel.java

```diff
@@ -546,6 +546,7 @@ void maybeLoadUI() {
     if (flutterAppFrameReady) {
       // We need to start by quering the inspector service to find out the
       // current state of the UI.
+      inspectorService.inferPubRootDirectoryIfNeeded();
```

Comment:
you need to wait for the pub root directory to be inferred before continuing.
Otherwise you will fetch the selection before you have the pub root directory.

Reply:
done